youtube_dl/extractor/telewebion.py

```diff
@@ -25,31 +25,26 @@ class TelewebionIE(InfoExtractor):
     def _real_extract(self, url):
         video_id = self._match_id(url)
 
-        secure_token = self._download_webpage(
-            'http://m.s2.telewebion.com/op/op?action=getSecurityToken', video_id)
-        episode_details = self._download_json(
-            'http://m.s2.telewebion.com/op/op', video_id,
-            query={'action': 'getEpisodeDetails', 'episode_id': video_id})
+        episode_details = self._download_json('https://gateway.telewebion.ir/kandoo/episode/getEpisodeDetail/?EpisodeId={}'.format(video_id), video_id)
+        episode_details = episode_details["body"]["queryEpisode"][0]
```

Comment:
1. Project convention: "" -> ''
2. Ancient Pythons can't count and need a number or keyword name inside {}
```suggestion
        episode_details = self._download_json('https://gateway.telewebion.ir/kandoo/episode/getEpisodeDetail/?EpisodeId={0}'.format(video_id), video_id)
        episode_details = episode_details['body']['queryEpisode'][0]
```